frontend/finance-tracker/src/components/ui/date-time-picker.tsx

```
@@ -0,0 +1,94 @@
+    modal = false,
+}: DateTimePickerProps) {
+    const [selectedTime, setSelectedTime] = React.useState(
+        date ? format(date, "HH:mm") : ""
+    );
+
+    const handleTimeChange = (time: string) => {
+        setSelectedTime(time);
+        if (date && time) {
+            const [hours, minutes] = time.split(":");
+            const newDate = new Date(date);
+            newDate.setHours(parseInt(hours, 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)